extension/scripts/features/faction-quick-items/ttFactionQuickItems.js

```diff
@@ -447,7 +447,7 @@
 
 				item.addEventListener("click", onItemClickQuickEdit);
 			}
-			for (const refill of document.findAll("#armoury-points .give[data-role='give']")) {
+			for (const refill of document.findAll("#armoury-points .give[data-role='give'], .give[data-role='refill']")) {

```

Comment:
Include the `#armoury-points` selector with the new part as well.

Reply:
Done.

For some reason during development I got an error doing this, but it sees to work fine now!